var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -60,13 +60,15 @@ class Trilinos(CMakePackage):
             description='Build python wrappers')
 
     # Build options
-    variant('complex', default=False,
+    variant('c',            default=False,
```

Comment:
Enabling this would require adding few conflicts, as CTrilinos has not been maintained for a while, and is not even part of the Trilinos for the last several releases.

Reply:
Ok, since its being phased out in recent releases I'll remove this variant.